### ISSUE TITLE
Recent change no longer includes cc when replying

### ISSUE THREAD

**Issue by stsquad** [djcb/mu#1544]
## Expected or desired behavior

Reply to all on a email list email should include CC list

## Actual behavior

Reply to all only replying to the original author

## Steps to reproduce

<R> <a> on a mailing list email

## Versions of mu, mu4e/emacs, operating system etc.

mu (master)
emacs (master)
Linux

## Any other detail

I'm trying to see where exactly the cc list gets dropped. It seems to only occur on list email where there are no name parts ((nil . "address@foo.com") (nil . "another@bar.com)) in the Cc list but it's a little hard to follow the logic. It doesn't look as though the special list handling code is being called anyway.

It looks like it gets dropped in:

    (mu4e~draft-header "Cc" (mu4e~draft-recipients-construct :cc origmsg)

via mu4e~draft-reply-construct-recipients but I'm finding it hard to follow at the moment.

**Comment by stsquad** [comment]
Ok it looks like mu4e~draft-reply-construct-recipient isn't passing reply-all down to the header generators:

```
           ;; otherwise...
           (concat
             (mu4e~draft-header "To" (mu4e~draft-recipients-construct :to origmsg))
-            (mu4e~draft-header "Cc" (mu4e~draft-recipients-construct :cc origmsg))))))))
+            (mu4e~draft-header "Cc" (mu4e~draft-recipients-construct :cc origmsg reply-all))))))))
 ```

**Comment by djcb** [comment]
Oh, silly mistake. Sorry about that thanks for the fix!
